metadata_extract/init_files.py

```diff
@@ -1,9 +1,10 @@
 # pylint: disable=missing-module-docstring
 import json
-
 from importlib.resources import files
 from typing import Type, Any
 
+from src.settings import get_settings
```

Comment:
This breaks the `run_on_file` script and makes it impossible to install metadata_extract as a module.
We want to keep the Meteor logic (`metadata_extract/`) separate from the FastAPI service (`src/`).
I think a good solution would be to pass the `LANGUAGES` list to the Meteor object (in `meteor.py`), either as a constructor argument or in a new setter method. Of course this means we have to rethink the way `InitFiles` is instantiated, the singleton construct may not work well with this new solution...

We also need to think of default behavior:

- if no `LANGUAGES` env variable is defined, do we take all languages or none? Currently the latter happens, and meteor does not work. Maybe not the best choice!
- `und` keywords should always be added.

Reply:
See commit [81a8f9e](https://github.com/NationalLibraryOfNorway/meteor/pull/9/commits/81a8f9ee0ed1c38b794593e0ef3a283a073597dc): Fixed `run_on_file` script breaking. Languages are now passed into Meteor class on initialization, and defaults to all languages present in the resource files if no languages are given. Some refactoring in `text.py` was necessary because static methods initializes before the Meteor object.

As for undetermined `und` language code, I'm not sure there should be separate logic to always apply this language code, but maybe it should be explicitly stated in the .env.example that `und` should always be in the language list. Thoughts?

Reply:
Module and script are fixed, perfect!
Good point regarding the common keywords, let's avoid hiding some implicit logic there. A comment in `.env.example` will do just fine.
But `mul` is probably a better choice than `und`.